collection.go

```diff
@@ -0,0 +1,11 @@
+package vt
+
+func PostCollectionFromRawText(client *Client, name, text string) (*Object, error) {
```

Comment:
I would leave this function out of `vt-go` at this moment, and implement it in `vt-cli` if necessary. We should think better if we want to commit to add some collections-specific API here, and which the API should look like.